modules/virt-runbook-vmstorageclasswarning.adoc

```diff
@@ -5,26 +5,26 @@
 // * virt/monitoring/virt-runbooks.adoc
```

Comment:
Did you read the comments? "Do not edit this module. It is generated with a script." Did anyone tell you about the runbook conversion script?

Reply:
Was that ^^ for @sbeskin-redhat ?

Reply:
Yes

Reply:
@apinnick 
Yes, i know about the script. However, my instructions from @ctomasko were as follows:
"You do not have to sync any scripts to update this runbook. You can open a PR and make this change. We need to update the downstream runbook to reflect the change that was made upstream."